Git-Workflow-Guidance.md

```diff
@@ -557,14 +666,54 @@ If you want a useful graphical summary of the log, try
  | merge | Merge changes from local repo to sandbox |
  | pull | Download changes to local sandbox (== fetch + merge) |
  | push | Upload changes |
- | rebase | like pull but updates the base version then replays local changes |
+ | rebase | like pull but updates the base version then replays local changes, same as pull --rebase |
  | remote [-v] [add] | 	Lists/Sets remote repositories in a sandbox |
  | rm | Removes a file from the repo |
  | status    | Provides summary of status of sandbox |
  | submodule | 	Many options, useful for working with submodules |
  | tag -a | Create a tag or list tags |
  | update | Pull updates into the local sandbox |
 
+### Further Information
+
+There are many git commands to diagnose the status of your sandbox.
+
+#### Status
+Status provides a summary of the status of your sandbox.  
+
+      git status
+
+will indicate which branch your sandbox is on and the changes that have been made.  This should be used often during development and before commits and pushes.
+
+#### Diff
+Diff shows code changes.
+
+      git diff
+
+If you also want to see the submodule diffs, use
+
+      git diff --submodule
+
+but it's important to remember that a commit to CICE will NOT commit the Icepack changes to Icepack.
+
+#### Log
+Log summarizes the commits and pushes in your sandbox up to the current state.  The first entry will provide the id of the current model version.
+
+      git log
+
+If you want a useful graphical summary of the log, try
+
+      git log --graph --oneline --decorate
```

Comment:
```suggestion
      git log --graph --oneline --decorate
      # or
      gitk
```

Some people might prefer a graphical tool (I don't personally use it but it's part of the Git distribution so we might as well mention it?)